application/templates/cms/edit_upload.html

```diff
@@ -10,7 +10,7 @@
 %}
 
 {% block bodyClasses %}rd_cms{% endblock %}
-{% block pageTitle %}{{ page_title("Edit download", error=(form.errors|length > 0) ) }}{% endblock %}
+{% block pageTitle %}{{ page_title("Edit source data", error=(form.errors|length > 0) ) }}{% endblock %}
```

Comment:
Not part of this PR I guess, but why on Earth do we have separate templates for adding and editing the data files?

Reply:
@TheDoubleK yeah, I spotted that too. One extends the other, but there's still quite a bit of duplication. I figured we can fix that at a later point though.

Reply:
Gonna resolve this question for now, as it doesn't relate to the content but to the code structure, which we can address later...